.github/workflows/slash-command.yml

```
@@ -0,0 +1,19 @@
+name: Slash Command
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  slashCommand:
+    name: Dispatch
+    runs-on: ubuntu-latest
+    steps:
+      uses: peter-evans/slash-command-dispatch@v3
+      with:
+        token: ${{ secrets.PAT }}
+        permission: admin
```

```suggestion
        permission: admin
```

Are we only allowing `admin` to do snapshot releases? I think any Shopifolk should be able to use it.